src/MICore/NativeMethods.cs

```diff
@@ -8,15 +8,15 @@ namespace MICore
 {
     internal class NativeMethods
     {
-        // TODO: It would be better to route these to the correct .so files directly rather than pasing through system.native. 
+        private const string Libc = "libc";
```

Comment:
These functions are called by LocalLinuxTransport, which is only used after checking the platform.  I will rename the class.